tst/com/amazon/ion/benchmark/testIntWithoutConstraint.isl

```diff
@@ -0,0 +1,4 @@
+type::{
+    name: Int,
+    type: int,
+}
```

Comment:
Should we also add tests for String, Symbol, Timestamp without any constraints except `type` constraint?
e.g.
```
type::{
    name: my_string,
    type: string,
} 
```

Reply:
> Should we also add tests for `String`, `Symbol`, `Timestamp` without any constraints except `type` constraint? e.g.
> 
> ```
> type::{
>     name: my_string,
>     type: string,
> } 
> ```

The reason we added test for `int` without any constraints except `type` constraint is that there is no specified constraints to narrow the `int` value. For the rest of data types, the functionalities of constraint `type` and other constraints do not work with each other(the data validation process has already covered these tests), so the additional tests for these data types might be duplicated with the exist unit tests.